src/core/qgsrendercontext.cpp

```diff
@@ -29,7 +29,8 @@
 #define INCH_TO_MM 25.4
 
 QgsRenderContext::QgsRenderContext()
-  : mFlags( DrawEditingInfo | UseAdvancedEffects | DrawSelection | UseRenderingOptimization )
+  : QgsTemporalRangeObject()
```

Comment:
same here -- we don't need to explicitly call the QgsTemporalRangeObject constructor here